src/components/Background/index.css

```diff
@@ -25,15 +25,28 @@
   }
 }
 
-.image {
+.stretch-to-fit {
   position: absolute !important;
-  top: 0;
-  right: 0;
-  bottom: 0;
-  left: 0;
+  top: -50%;
+  left: -50%;
 
-  width: 100%;
-  height: 100%;
+  width: 200%;
+  height: 200%;
+
+  @supports not (object-fit: cover) {
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
```

Comment:
If you don't support `object-fit: cover` this line is useless right?

Reply:
Fixed